zlp-scheduler/app/views/admin/view_result.html.haml

```diff
@@ -11,37 +11,40 @@
         %th#special_table_header Day
         %th#special_table_header Time
         %th#special_table_header Option
-      %tr
-        %td#special_table_data Monday
-        %td#special_table_data 1:45 PM - 3:45 PM
-        %td#special_table_data
-          %form
-            %button.choosebtn{:formaction => "view_term_admin"} Choose
+      -@results.each do |result|
+        %tr
+          %td#special_table_data #{result.day} 
+          %td#special_table_data #{result.time}
+          %td#special_table_data
+            %form
+              %button.choosebtn{:formaction => "view_conflicts"} Choose
+
       %tr
         %td#special_table_data Wednesday
         %td#special_table_data 1:45 PM - 3:45 PM
         %td#special_table_data
           %form
-            %button.choosebtn{:formaction => "view_term_admin"} Choose
+            %button.choosebtn{:formaction => "view_conflicts"} Choose
       %tr
         %td#special_table_data Friday
         %td#special_table_data 1:45 PM - 3:45 PM
         %td#special_table_data
           %form
-            %button.choosebtn{:formaction => "view_term_admin"} Choose
+            %button.choosebtn{:formaction => "view_conflicts"} Choose
```

Comment:
Can we get rid of the hard-coded table entries? And just have the ones from the database?

Reply:
Both here and below, in the conflicts table.